components/alert/alert-toast.js

```diff
@@ -257,29 +272,50 @@ class AlertToast extends LitElement {
 		clearTimeout(this._setTimeoutId);
 	}
 
-	_handleAlertClose(e) {
-		if (e?.target === this || !this.open) return;
+	_handleButtonPress(e) {
+		e.stopPropagation();
+		this.dispatchEvent(new CustomEvent('d2l-alert-toast-button-press'));
+	}
+
+	_handlePageResize(e) {
+		this._smallWidth = e.matches;
+	}
 
-		const containerBottom = parseFloat(getComputedStyle(this._innerContainer).getPropertyValue('bottom'));
-		const closingContainerBottom = e.detail.bottom;
-		if (closingContainerBottom > containerBottom) return; // closing alert is above this alert, no need to adjust bottom spacing
+	_handleResize() {
+		const newHeight = this._innerContainer.offsetHeight;
```

Comment:
I think that could work. The heights seem to be the same. The bottoms are different (`getComputedStyle` returns relative to the bottom vs `getBoundingClientRect` is relative to the top) but since we're just using that to compare positions on the screen it would still be usable.